README.rst

```diff
@@ -48,21 +64,25 @@ Include the following in your buildout configuration to create an `i18ndude`_
 helper script to update the po files of your product::
 
     [buildout]
-    parts =
-        ...
+    extends =
+        https://raw.github.com/collective/buildout.plonetest/master/test-5.x.cfg
+        https://raw.github.com/collective/buildout.plonetest/master/qa.cfg
+    package-name = plone.app.foo
+    package-extras = [test]
+    parts+=
+        i18ndude
         rebuild_i18n-sh
 
-    [rebuild_i18n-sh]
-    recipe = collective.recipe.template
-    url = https://raw.github.com/collective/buildout.plonetest/master/templates/rebuild_i18n.sh.in
-    output = ${buildout:directory}/bin/rebuild_i18n.sh
-    mode = 755
-
-After running 'bin/buildout' you will find a 'bin/rebuild_i18n.sh'; run the
+After running ``bin/buildout`` you will find a ``bin/rebuild_i18n.sh``; run the
 script and the po files will be updated.
 
-Domain name is taken from the ${buildout:package-name} variable; Plone domain
-is also included.
+Domain name is taken from the ``${buildout:package-name}`` variable.
+The plone domain is also included.
+
+.. ATTENTION::
+   Originally, the ``rebuild_i18n-sh`` part used a `filesystem template <https://github.com/collective/buildout.plonetest/blob/master/templates/rebuild_i18n.sh.in>`_.
```

Comment:
I think this file could be removed.

Reply:
Eventually the file could be removed, but current master is still pointing to it, so I think we should leave it in for now.